### DISCUSSION TITLE
The initial login details for the /admin, /c and the /a routes

### DISCUSSION THREAD

**Discussion by Muasya** [N-John/mikrotik_hermes#8] [Q&A] [answered]
I have run this project, but going through the source code to figure out the initial user authentication values has not helped. The SQLite db does not include those either. Are the /admin, /c and the /a routes login creds some defaults values or is there a way to add them for testing?

**Comment by N-John** [comment, marked as answer]
Hello @Muasya . Thank you for waiting. 

 For the cli program, when you run the code for the first time, a variables.txt file will be created to hold the Db name, log file name, router Ip, and router login credentials. You get the chance to edit these variables as per your desire before pressing "Y" for the code to proceed. From there the program will login to the router and retrieve Pppoe and hotspot users already set on it (if any). Once the Database has been created, you can freely add/edit any values for testing and RL.

For the web, the default Django SQLite will hold the credentials.

**Comment by N-John** [comment]
Kindly note that I had gone on hiatus before finishing the Hermes web. I am currently testing it before pushing. It will contain more functions compared to the cli for automation, system admin and staff management, and also provide users with an account website where they can manage some account actions from their side so as to relief the admin. I'll have it up by the end of the month.

Thanks
